tests/utils.go

```diff
@@ -2613,6 +2613,30 @@ func AddSecretDiskWithCustomLabel(vmi *v1.VirtualMachineInstance, secretName str
 	})
 }
 
+func AddDownwardAPIVolumeWithLabel(vmi *v1.VirtualMachineInstance, volumeName string, labels map[string]string) {
+	vmi.ObjectMeta.Labels = labels
```

Comment:
nit: best to set the labels directly in the test on the VMI. This has  the side-effect or replacing any labels which one would want to set on the VMI. I would not expect that when I want to add one or more downward api volumes with this function.

Reply:
@rmohr Oh I was planning to change this to add specific label, instead of override all. Left it there by mistake, will fix

Reply:
@rmohr Done. Setting the testing label in the test and not the helper. The helper only add the volume & disk